metrics: Update machine learning documentation (#7360)
kata-containers/kata-containers: tests/metrics/machine_learning/README.md

Kata Containers provides a series of performance tests using the
TensorFlow reference benchmarks (tf_cnn_benchmarks).
The tf_cnn_benchmarks containers TensorFlow implementations of several
popular convolutional models https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks.

> Currently the Kata Containers TensorFlow benchmark includes test for

Reviewer: ```suggestion
Currently the TensorFlow benchmark on Kata Containers includes
```

> Currently the Kata Containers TensorFlow benchmark includes test for

Reviewer: ```suggestion
Currently the TensorFlow benchmark on Kata Containers includes
```
